app/static/pages/volunteer/VolunteerProfilesService.js

```diff
@@ -41,23 +41,26 @@ oppiaFoundationWebsite.factory('VolunteerProfilesService', [
     };
     var volunteersByCategory = {};
     Object.keys(VOLUNTEER_INFO).forEach(function(volunteerCategory) {
+
+      console.log(volunteerCategory);
+      
       volunteersByCategory[volunteerCategory] = getRandomProfiles(
         VOLUNTEER_INFO[volunteerCategory]);
     });
 
     var VolunteerProfilesService = {
-      getDesignProfiles: function() {
-        return volunteersByCategory.design;
+      getArtistProfiles: function() {
+        return volunteersByCategory.art;
```

Comment:
It's `volunteersByCategory.artist` like you defined in `VolunteerInfoDefinition.js`

Reply:
(adjusted to be art instead of artist in all references)